dotnet/src/webdriver/Remote/ResponseLoggerInterceptor.cs

```diff
@@ -0,0 +1,58 @@
+// <copyright file="ResponseLoggerInterceptor.cs" company="WebDriver Committers">
```

Comment:
Usually all classes should be in separated files. I have no strong opinion, probably this class may live in `HttpCommandExecutor` file. It is discussable. I would put this class even into `HttpCommandExecutor` class as iner class and name it `DiagnosticsHttpHanddler`. And for sure make it as `internal`.

Reply:
Agree! That’s totally make sense.